tests/docs/partials/invalid_content_type_headers.rst

```diff
@@ -0,0 +1,59 @@
+Content Type Headers
```

Comment:
I think we need to swap these over to just being bold text instead - it really messes up the header sizes across lots of the docs